tox.ini

```diff
@@ -0,0 +1,13 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = True
+envlist = py35
+
+[testenv]
+deps =
+    -rrequirements.txt
+commands = py.test
```

Comment:
Hmm, okay, it seems to work fine with recent pip versions for me - but I remember having this problem earlier.